src/SFA.DAS.QnA.Api.UnitTests/ApplicationDataControllerTests/WhenGettingApplicationData.cs

```diff
@@ -0,0 +1,90 @@
+﻿using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using NSubstitute;
+using NUnit.Framework;
+using SFA.DAS.QnA.Api.Types;
+using SFA.DAS.QnA.Application.Queries.ApplicationData.GetApplicationData;
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.QnA.Api.UnitTests.ApplicationDataControllerTests
+{
+    [TestFixture]
+    public class WhenGettingApplicationData
+    {
+        [Test]
+        public async Task And_MediatorCallFails_ThenNotFoundResultReturned()
+        {
+            var mediator = Substitute.For<IMediator>();
+            var controller = new Controllers.ApplicationDataController(mediator);
+            mediator.Send(Arg.Any<GetApplicationDataRequest>()).Returns(new HandlerResponse<string>() { Success = false, Message = "Application does not exist." });
+
+            var result = await controller.Get(Guid.NewGuid());
+
+            result.Result.Should().BeOfType<NotFoundObjectResult>();
+        }
+
+        [Test]
+        public async Task And_MediatorCallIsSuccessful_ThenOkObjectResultReturned()
+        {
+            var mediator = Substitute.For<IMediator>();
+            var controller = new Controllers.ApplicationDataController(mediator);
+            var applicationData = File.ReadAllText("ApplicationDataControllerTests/test.json");
```

Comment:
We can remove this and make the mediator return what it needs to trigger the controller's happy path, like you're doing for the Post tests